autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/ExpandedCourseCard.tsx

```diff
@@ -28,6 +28,19 @@ const ExpandedCourseCard: React.FC<ExpandedCourseCardProps> = ({
   const dispatch = useDispatch();
   const { course, customizationLevel } = courseCardOptions;
 
+  const [options, setOptions] = React.useState([]);
+  const [inputValue, setInputValue] = React.useState('');
+
+  function getAutocomplete(text: string): void {
+    fetch(`api/course/search?search=${text}&term=201931`).then(
+      (res) => res.json(),
+    ).then((result) => {
+      if (result.results) {
+        setOptions(result.results);
+      }
```

Comment:
What exactly is the point of this check?

Reply:
Uh typical null check? I think it was needed at one point but idk if it still is or not lol